deploy/deployment-manager/service_account.py

```diff
@@ -14,7 +14,7 @@ def generate_config(context):
         "name": sa_name,
         "type": "iam.v1.serviceAccount",
         "properties": {
-            "accountId": sa_name,
+            "accountId": f"{sa_name}-id",
```

Comment:
i'm not entirely sure why this is needed in addition to `dependsOn`, but i ran into some flakiness when deploying the service account, this seems to fix it.